models/quality_measures/intermediate/nqf2372_breast_cancer_screening/quality_measures__int_nqf2372_exclude_dementia.sql

```diff
@@ -351,10 +122,10 @@ with denominator as (
             on patients_with_frailty.patient_id = pharmacy_claim_exclusions.patient_id
     where (
         pharmacy_claim_exclusions.dispensing_date
-            between patients_with_frailty.performance_period_begin
+            between {{ dbt.dateadd(datepart="year", interval=-1, from_date_or_timestamp="patients_with_frailty.performance_period_begin") }}
             and patients_with_frailty.performance_period_end
         or pharmacy_claim_exclusions.paid_date
-            between patients_with_frailty.performance_period_begin
+            between {{ dbt.dateadd(datepart="year", interval=-1, from_date_or_timestamp="patients_with_frailty.performance_period_begin") }}
```

Comment:
Found another bug. This exclusion applies for the measurement period and the year prior.

> Patients 66 years of age and older with at least one
claim/encounter for frailty during the measurement period
AND a dispensed medication for dementia during the
measurement period or the year prior to the
measurement period